packages/terra-modal/src/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: $terra-white;
+  border-radius: 5px;
+  bottom: 10px;
+  display: block;
+  left: 10px;
+  max-height: 100%;
+  outline: none;
+  overflow: hidden;
+  position: fixed;
+  right: 10px;
```

Comment:
Update this to use to position-end mixin: Update this to use the position-start mixin: https://github.com/cerner/terra-core/tree/master/packages/terra-mixins/docs#terra-position-start--terra-position-end